shared/queues/queues.ts

```diff
@@ -0,0 +1,64 @@
+import { v4 } from 'uuid';
+import { Queue } from 'bullmq';
+import { redisConnection } from '../config'; // Adjust the import based on your structure
+
+export const QUEUE_NAMES = {
```

Comment:
since there is just one queue. do we need this? YAGNI principle

Reply:
i missed this one, was thinking were to place it